modules/api/src/main/java/io/fluo/api/types/TypeLayer.java

```diff
@@ -198,17 +199,19 @@
  * 
  * <pre>
  * {@code
- *   //pretend this method has curly braces, getting javadoc to work w/ less than and curly braces is too hard
+ *   // pretend this method has curly braces, getting javadoc to work w/ less than and
```

Comment:
Did you look at how this javadoc renders?  If not I can take a look .  I remeber wriring all this code in javadoc being tricky.